draft-ietf-httpbis-resumable-upload.md

```diff
@@ -284,25 +275,27 @@ Client implementations of draft versions of the protocol MUST ignore a `104 (Upl
 
 The reason both the client and the server are sending and checking the draft version is to ensure that implementations of the final RFC will not accidentally interop with draft implementations, as they will not check the existence of the `Upload-Draft-Interop-Version` header field.
 
-# Offset Retrieving Procedure {#offset-retrieving}
+# Offset Retrieval {#offset-retrieving}
 
-If an upload is interrupted, the client MAY attempt to fetch the offset of the incomplete upload by sending a `HEAD` request to the upload URL, as obtained from the Upload Creation Procedure ({{upload-creation}}). The client MUST NOT initiate this procedure without the knowledge of server support.
+If an upload is interrupted, the client MAY attempt to fetch the offset of the incomplete upload by sending a `HEAD` request to the upload resource.
```

Comment:
Removal of a MUST NOT here, so flagging.

Reply:
How can there be such a resource if the server doesn’t support this feature?

Reply:
Is that specific to this pr?

Reply:
I agree that this should be OK to remove now as @guoye-zhang explained.